server/runtime_go_nakama.go

```diff
@@ -2524,6 +3268,12 @@ func (n *RuntimeGoNakamaModule) ChannelMessageUpdate(ctx context.Context, channe
 	return ChannelMessageUpdate(ctx, n.logger, n.db, n.router, channelIdToStreamResult.Stream, channelId, messageId, contentStr, senderId, senderUsername, persist)
 }
 
+// @summary Create a chat channel.
```

Comment:
Helper function to create an channel identifier that can be used for various other runtime calls; Note, this does NOT create a channel.

Target: Can be the room name, group identifier, or another username

channelId: The generated ID that represents a channel.